tests/API/Helpers/State/SessionStateHandlerTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+namespace Auth0\Tests\Api\Helpers\State;
+
+use Auth0\SDK\API\Helpers\State\SessionStateHandler;
+use Auth0\SDK\Store\SessionStore;
+
+/**
+ * Class SessionStateHandlerTest
+ *
+ * @package Auth0\Tests\Api\Helpers\State
+ */
+class SessionStateHandlerTest extends \PHPUnit_Framework_TestCase
+{
+
+    /**
+     * Storage engine to use.
+     *
+     * @var SessionStore
+     */
+    private $store;
+
+    /**
+     * State handler to use.
+     *
+     * @var SessionStateHandler
+     */
+    private $state;
+
+    /**
+     * SessionStateHandlerTest constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+
+        // Suppress header sent error
+        @$this->store = new SessionStore();
+        $this->state = new SessionStateHandler($this->store);
+    }
+
+    /**
+     * Test that state is stored and retrieved properly.
+     */
+    public function testStateStoredCorrectly()
+    {
+        $uniqid = uniqid();
+        $this->state->store($uniqid);
+        $this->assertEquals($uniqid, $this->store->get(SessionStateHandler::STATE_NAME));
+    }
+
+    /**
+     * Test that the state is being issued correctly.
+     */
+    public function testStateIssuedCorrectly()
+    {
+        $state_issued = $this->state->issue();
```

Comment:
Issue() stores using the default name, right?

Reply:
yes